packages/reports/tests/dummy/config/environment.js

```diff
@@ -64,11 +64,24 @@ module.exports = function (environment) {
           type: 'elide',
         },
         {
-          name: 'elideTwo',
-          displayName: 'Elide Two',
-          description: 'Elide Two Description',
+          name: 'elideTwoDefault',
+          displayName: 'Elide Two Default',
+          isHidden: true,
```

Comment:
might be a little confusing that `isHidden` is true at the root level yet some of the namespaces are still shown.
but it's either a flag (`hideDefaultNamespace` is probably too elide-specific) or flatten everything and add namespace id for each entry
anyway I guess it's good for what we need now 👍 and we can refactor in the future